### ISSUE TITLE
Koda prod 500 or unresponsive

### ISSUE THREAD

**Issue by hassnian** [kodadot/nft-gallery#10797]
### Is there an existing issue?

- [X] I have searched the existing issues

### What happened?

not the first time that I've gotten this, don't know if it's my connection but sometimes it takes a lot of time to load or multiple retries, just with koda domains other websites work fine. 

Anyone else experiencing the same?

cc @preschian 

![CleanShot 2024-08-12 at 14 55 00@2x](https://github.com/user-attachments/assets/2b150d89-fc70-4f32-8d3a-c313d74f1fa6)

### Please reproduce in steps

-

### Expected Behavior

-

### Are you logged in?

None

### Screenshots

-

### Payment link for reward

https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by preschian** [comment]
> Anyone else experiencing the same?

Yes, I also rarely face this issue. Reduce js size probably could help resolve the issue

**Comment by preschian** [comment]
additional context, this is nothing with the domain. 

<img width="200" alt="Screenshot 2024-08-13 at 10 30 13" src="https://github.com/user-attachments/assets/90ead17c-f973-4823-9fa7-2d742f603958">

maybe anyone want to take a look this one `https://github.com/vitejs/vite/issues/11804`

**Comment by hassnian** [comment]
>  this is nothing with the domain.

bad wording I meant to say the koda deployed app

**Comment by hassnian** [comment]
![CleanShot 2024-08-14 at 12 20 15@2x](https://github.com/user-attachments/assets/80220615-f65a-44ae-9152-fd958fb18f3b)
